extensions.py

```diff
@@ -0,0 +1,24 @@
+#get user input
+text = input("File name: ")
+#convert input to lower with .lower string method
+text1 = text.lower()
+#create function that involves branching/ produces desired outcome w/ paramater
+def mediatype(user):
+    if ".jpeg" in user:
```

Comment:
Could you use a case statement here?

https://www.geeksforgeeks.org/python-match-case-statement/

Reply:
@sosaeio @sosaeio Thank you! Both suggestions could end in better code optimization.  I will make the necessary revision as soon as possible.